pilot/pkg/model/service.go

```diff
@@ -861,7 +879,8 @@ func (i *WorkloadInfo) Clone() *WorkloadInfo {
 
 func (i WorkloadInfo) ResourceName() string {
 	ii, _ := netip.AddrFromSlice(i.Address)
-	return ii.String()
+	name := i.Network + "/" + ii.String()
```

Comment:
nit; can we add the same TODO as we did for https://github.com/istio/istio/pull/44199/files#r1195255541

Reply:
doen